client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/style.scss

```diff
@@ -0,0 +1,27 @@
+.site-migration-instructions {
+	.site-migration-instructions__list {
+		li {
+			margin-bottom: 0.5rem;
+		}
+	}
+
+	.step-container__header p {
+		margin-top: 1.5rem;
+	}
+
+	.site-migration-instructions__migration-key {
+		display: flex;
+		justify-content: space-between;
+		margin-top: 0.5rem;
+		max-width: 100%;
+	}
+
+	.site-migration-instructions__migration-key .site-migration-instructions__key {
+		background: var(--color-neutral-0);
+		display: inline-block;
+		margin-right: 0.75rem;
+		padding: 0.75rem 1rem;
+		width: 400px;
+		overflow-y: scroll;
```

Comment:
I think this should be `overflow-x` instead of `overflow-y`. This is what I see now:
<img width="586" alt="image" src="https://github.com/Automattic/wp-calypso/assets/375980/5201658d-9611-40ff-8959-3005263e10fb">
vs what I see if I switch to `overflow-x`
<img width="602" alt="image" src="https://github.com/Automattic/wp-calypso/assets/375980/03c9227d-5f68-4a56-aa98-28d0e55b827f">